docs/user-guide/class-b-beaconing.md

```diff
@@ -0,0 +1,171 @@
+# Class-B: Beacon locking
+
+This doc describes the process of locking a Class-B arduino device to a beaconing signal issued from the basic station and reading the GPS coordinates transmitted by that beacon.
+
+## Network Server
+The LoRaWAN network server (`LoraWanNetworkSrvModule`) doesn't need any special configuration beyond the usual required launch settings.
+
+> **Note**: For the time being the network server code required to support Class B beaconing lives in the branch `feature/add-beaconing`
```

Comment:
Not sure I would write this, this will get outdated as soon as we finish the tests and we might forget to update it